build/tasks/release.js

```diff
@@ -49,6 +49,7 @@ gulp.task('bundle', function () {
   var bundles = [
     'app/cart/cart.component',
     'app/checkout/checkout.component',
+    'app/thank-you/thank-you.component',
```

Comment:
It kinda looks like I started doing all the services in camelCase and all the components in kebab-case. I don't know why I did that. I must have made the first ones that way and just copied it... Opps